docs/code/IDataViewTypeSystem.md

```diff
@@ -136,7 +137,7 @@ level, this dimensionality information indicates the length of the vector
 type. A length of zero means that the vector type is variable length, that is,
 different values may have different lengths. Additional detail of vector types
 is in a subsequent section. Vector types are instances of the sealed class
-`VectorType`, which derives from `ColumnType`.
+`VectorDataViewType`, which derives from `ColumnDataViewType`.
 
 This document uses convenient shorthand for standard types:
```

Comment:
Maybe it will be a lot of work, but it would be nice if this document wouldn't use these "convenient shorthand". Especially since we have removed these names from our public API.

Reply:
I agree, and I spent a few hours at it, but it's so extensive and the language gets so long, that I just couldn't keep it up.